pkg/drivers/qemu/qemu.go

```diff
@@ -412,14 +413,19 @@ func (d *Driver) Start() error {
 	}
 
 	// hardware acceleration is important, it increases performance by 10x
-	if runtime.GOOS == "darwin" {
+	var accel string
```

Comment:
lets move this logic to a helper function to address the complexity lint

something like SelectAccelMode()